doc/source/ray-observability/ray-logging.rst

```diff
@@ -331,7 +341,8 @@ If using the Ray cluster launcher, you would specify this environment variable i
 
 When connecting to the cluster, be sure to set the environment variable before connecting:
 
-.. code-block:: python
+.. testcode::
+    :skipif: True
 
     os.environ["RAY_LOG_TO_STDERR"] = "1"
     ray.init(address="auto")
```

Comment:
Would it be possible to test this?

Reply:
Probably hard: we need to start a ray cluster via `ray start` first.